easy-to-use-api/klips-chart-api/src/components/Docs/index.ts

```diff
@@ -18,7 +18,7 @@ export class ChartAPIDocs {
         const docElement: HTMLElement | null = document.querySelector('#url-params-documentation');
         if (docElement) {
             let table = '';
-            for (let i = 0; i < this.params.length; i++) {
+            for (let i = 0; i < this.params[0].length; i++) {
```

Comment:
Could you please briefly explain why this change gets necessary or better why is now iterated over `params[0]` instead of `params`? Thanks in advance...

Reply:
params was a list of arrays defined in the content.json file. I needed the number of parameters (rows in the table) here. I now changed the structure of the params object, I hope, it is more clear that way?